composer.json

```diff
@@ -1,5 +1,16 @@
 {
+    "name": "lndrest/composer",
+    "description": "An lnd class",
+    "type": "composer-plugin",
```

Comment:
I do not think this is a "compser-plugin"

from the documentation: 

"composer-plugin: A package of type composer-plugin may provide an installer for other packages that have a custom type. Read more in the [dedicated article](https://getcomposer.org/doc/articles/custom-installers.md)."

I think this is more like a library: https://getcomposer.org/doc/04-schema.md#type